packages/discord.js/src/structures/User.js

```diff
@@ -246,6 +270,19 @@ class User extends Base {
     return this.banner && this.client.rest.cdn.banner(this.id, this.banner, options);
   }
 
+  /**
+   * A link to the user's guild tag badge.
+   *
+   * @param {ImageURLOptions} [options={}] Options for the image URL
+   * @returns {?string}
+   */
+  guildTagBadgeURL(options = {}) {
+    return (
+      this.primaryGuild?.badge &&
+      this.client.rest.cdn.guildTagBadge(this.primaryGuild.identityGuildId, this.primaryGuild.badge, options)
+    );
```

Comment:
This would return `undefined` otherwise.

```suggestion
    return this.primaryGuild?.badge
      ? this.client.rest.cdn.guildTagBadge(this.primaryGuild.identityGuildId, this.primaryGuild.badge, options)
      : null;
```